core/src/scene/importer.cpp

```diff
@@ -126,18 +153,64 @@ bool nodeIsTextureUrl(const Node& node, const Node& textures) {
     return true;
 }
 
-void Importer::resolveSceneUrls(const std::shared_ptr<Platform>& platform, Node& root,
-        const Url& base) {
+void  Importer::createSceneAsset(const std::shared_ptr<Platform>& platform, std::shared_ptr<Scene>& scene,
+        const Url& resolvedUrl, const Url& relativeUrl, const Url& base) {
+
+    auto& sceneAssets = scene->sceneAssets();
+    auto& resolvedStr = resolvedUrl.string();
+    auto& baseStr = base.string();
+    auto relativeStr = relativeUrl.string();
+
+    if (sceneAssets.find(resolvedStr) != sceneAssets.end()) { return; }
+
+    // needed to find apt file inside zipBundle
+    if (relativeStr[0] == '/') { relativeStr.erase(relativeStr.begin()); }
```

Comment:
Ahh.. I had it done properly before (refer commit history). Reverted back and addressed in: a81f3d8